content/docs/user-guide/what-is-dvc.md

```diff
@@ -1,16 +1,18 @@
 # What Is DVC?
 
-**Data Version Control** is a new type of data versioning, workflow and
-experiment management software, that builds upon [Git](https://git-scm.com/)
-(although it can work stand-alone). DVC reduces the gap between established
-engineering tool sets and data science needs, allowing users to take advantage
-of new [features](#core-features) while reusing existing skills and intuition.
+**Data Version Control** is a new type of data versioning, workflow and machine
+learning experiment management software that builds upon
+[Git](https://git-scm.com/) (although it can work stand-alone). DVC reduces the
+gap between established engineering tool sets and data science needs, allowing
+users to take advantage of new [features](#core-features) while reusing existing
+skills and intuition.
 
-![](/img/reproducibility.png) _DVC codifies data and ML experiments_
+![](/img/reproducibility.png) _DVC codifies data and machine learning
+experiments_
 
-Data science experiment sharing and collaboration can be done through a regular
-Git flow (commits, branching, pull requests, etc.), the same way it works for
-software engineers.
+Using DVC, data scientists can apply a regular Git flow to ML project sharing
+and collaboration (commits, branching, pull requests, etc.), the same way it
+works for software engineers.
```

Comment:
I see "ML" is used here. What are the guidelines to decide whether to use the full term or an abbreviation?

Also, this change kind of modifies the meaning of the text. We were previously talking about sharing data science experiments, now we're talking about data scientists sharing ML projects. Can we avoid semantic changes if the goal of the PR is mainly to insert a term? Unless you want to suggest a better writing of course, but not sure that was your intention.

Reply:
> I see "ML" is used here. What are the guidelines to decide whether to use the full term or an abbreviation?

Clarity can remain the priority. We just want both abbreviated and unabbreviated forms on a document level, including the combined usage variants we're pursuing for SEO.
 
> Can we avoid semantic changes if the goal of the PR is mainly to insert a term? 

Yes, of course. I was trying to get "ML" as close to "flow" as possible to address the competitive term "MLflow". This is a fairly aggressive approach — we don't need to go this far. For this initial PR I wanted added examples both less and more aggressive so we could decide on what is acceptable and what is "too far". 😉

Reply:
> I was trying to get "ML" as close to "flow" as possible to address the competitive term "MLflow".

Oh, interesting. In that case we could consider this change. What could that achieve in terms of SEO? Cc @shcheklein 

BTW you just removed "ML" now so we don't get the new term benefit and lose the original meaning. I'd try reverting it first, and then rewrite it again 🙂

Reply:
OK, added the main term I'm looking to add in an expanded version of this paragraph. I think the addition fulfills the primary need for this doc so we can leave the ML + flow consideration for elsewhere.